README.md

```diff
@@ -29,19 +29,35 @@ The framework has the following functionalities:
 
 ### Add Mock Data Using the Apple Health App
 
-Use the `XCTestCase`'s `exitAppAndOpenHealth(_: HealthAppDataType) throws` function passing in an `HealthAppDataType` instance to add mock data using the Apple Health app:
+Use the `XCTestCase.launchAndAddSample(healthApp:_:) throws` function passing in an `NewHealthSampleInput` instance to add mock data using the Apple Health app:
```

Comment:
Unfortunately we have a bit of a duplication in all our repos regarding the READMEs and DocC landing pages; we would need to make all of the same changes to the DocC main page: https://github.com/StanfordBDHG/XCTHealthKit/blob/main/Sources/XCTHealthKit/XCTHealthKit.docc/XCTHealthKit.md

Reply:
done 👍